### PR TITLE
Simplify aws policy by adding cortex prefix to queue name

### PR DESCRIPTION
**Pull request by vishalbollu**
---

checklist:

- [x] run `make test` and `make lint`
- [x] test manually (i.e. build/push all images, restart operator, and re-deploy APIs)

### CODE REVIEW

File: docs/clusters/aws/auth.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ _NOTE: The policy created during `cortex cluster up` will automatically be delet
 		{
 			"Effect": "Allow",
 			"Action": "sqs:*",
-			"Resource": "arn:aws:sqs:{{ .Region }}:{{ .AccountID }}:{{ .SQSPrefix }}*"
+			"Resource": "arn:aws:sqs:{{ .Region }}:{{ .AccountID }}:cortex-*"
```
Comment (miguelvr) [inline docs/clusters/aws/auth.md:80]: nit: convert to const

File: pkg/types/clusterconfig/aws_policy.go [hunk 2]
```diff
@@ -53,7 +53,7 @@ var _cortexPolicy = `
 		{
 			"Effect": "Allow",
 			"Action": "sqs:*",
-			"Resource": "arn:aws:sqs:{{ .Region }}:{{ .AccountID }}:{{ .SQSPrefix }}*"
+			"Resource": "arn:aws:sqs:{{ .Region }}:{{ .AccountID }}:cortex-*"
```
Comment (miguelvr) [inline pkg/types/clusterconfig/aws_policy.go:56]: nit: convert to const

**Review by deliahu — APPROVED** [review]

**Review by RobertLucian — APPROVED** [review]

File: docs/clusters/aws/auth.md (hunk 1 above)
Comment (vishalbollu) [reply docs/clusters/aws/auth.md:80]: What do you mean here?

File: pkg/operator/resources/job/batchapi/queue.go [hunk 3]
```diff
@@ -35,8 +35,9 @@ func apiQueueNamePrefix(apiName string) string {
 	return config.CoreConfig.SQSNamePrefix() + apiName + "-"
 }
 
-// QueueName is <hash of cluster name>-<api_name>-<job_id>.fifo
+// QueueName is cortex-<hash of cluster name>-<api_name>-<job_id>.fifo
 func getJobQueueName(jobKey spec.JobKey) string {
+	fmt.Println(apiQueueNamePrefix(jobKey.APIName) + jobKey.ID + ".fifo")
```
[the remark this answers is not in the document]
Comment (vishalbollu) [reply pkg/operator/resources/job/batchapi/queue.go:40]: removed

File: docs/clusters/aws/auth.md (hunk 1 above)
Comment (miguelvr) [reply docs/clusters/aws/auth.md:80]: The cortex- prefix

**Pull request merged**